docs/UserGuide.adoc

```diff
@@ -234,11 +323,34 @@ Address book data are saved in the hard disk automatically after any command tha
 There is no need to save manually.
 
 // tag::dataencryption[]
-=== Encrypting data files `[coming in v2.0]`
+=== Encrypting data files : `password` (coming in v1.3)
+
+Changes the password used to encrypt the address book. +
+Format: `password PASSWORD`
+
+****
+* Encrypts the address book using the specified PASSWORD.
+* Passwords are case-sensitive.
+* The password string can only contain characters in the UTF-8 format.
+****
+
+Examples:
+
+* `password test` +
+Encrypts `addressbook.xml` with "test" as the key.
 
-_{explain how the user can enable/disable data encryption}_
 // end::dataencryption[]
 
+=== Sorting all persons by module code: `sortCode` (coming in v2.0)
+
+Sorts all persons in the addressbook by module code in the order of A to Z. +
+Format: `sortCode`
+
+****
+* Displays full name of persons, arranged in the order of the modules they are taking.
+* There may by multiple entries for each person.
+****
+
```

Comment:
Personally I find the section on `sortCode` hard to understand. I cannot quite visualise what you mean here. Maybe adding an example can help...